code/index.html

```diff
@@ -2,21 +2,136 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
+    <link rel="icon" type="image/png" href="./pic/favicon-16x16.png" sizes="16x16" />
+    <title>ONE 2 TREE NEWS</title>
+    <link rel="stylesheet" href="./style.css"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
-    <header>
+
+    <header class="header">
+      <div class="logo-and-text">
+        <img class="logo" src="./pic/logo.png" alt="logo for the tree news">
+        <h1 class="header-text">ONE 2 TREE NEWS</h1>
+    </div>
+      <nav>
+        <a href="#">Read more</a>
+        <a href="#">Members</a>
+        <a href="#">Save a tree</a>
+        <a href="http://technigo.io">About us</a>
+      </nav>
       <!-- Some kind of logo and navbar -->
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+    <main class="main">
+      
+      <section class="big-news">
+       <!-- The big news content -->
+       <img class="big-news-img" src="./pic/wood_header.jpeg" alt="autumn wood">
+       <div class="big-news-text">
+         <h1> World's Oldest Living Tree </h1>
+        <p>-- 9550 years old -- Discovered In Sweden Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum. </p>
+      </div>
+      </section>
 
-    <section class="">
-      <!-- The other news  -->
+     <section class="grid-news">
+     
+      <!-- Grids x 6 -->
+       <!-- Grid1 -->
+      <div class=grid-content>
+        <img class="grid-picture" src="./pic/grid-river.jpg" alt="">
```

Comment:
You could fill in the alt for the picture here.

Reply:
Thanks I will